agent/README.md

```diff
@@ -128,7 +128,7 @@ echo `git rev-parse --abbrev-ref HEAD`:`git --no-pager log -n1 --format="%ct000"
 - `teamscale-commit-manifest-jar` As an alternative to `teamscale-commit` the agent accepts values supplied via 
```

Comment:
(Belongs to line 97, but I cannot attach a comment there)
We should mention that this is optional and if not specified we expect the teamscale.project to be specified in the git.properties